notebooks/kubeflow_pipelines/cicd/labs/trainer_image/train.py

```diff
@@ -43,25 +43,13 @@ def train_evaluate(
     if not hptune:
         df_train = pd.concat([df_train, df_validation])
 
-    numeric_features = [
-        "Elevation",
-        "Aspect",
-        "Slope",
-        "Horizontal_Distance_To_Hydrology",
-        "Vertical_Distance_To_Hydrology",
-        "Horizontal_Distance_To_Roadways",
-        "Hillshade_9am",
-        "Hillshade_Noon",
-        "Hillshade_3pm",
-        "Horizontal_Distance_To_Fire_Points",
-    ]
-
-    categorical_features = ["Wilderness_Area", "Soil_Type"]
+    numeric_feature_indexes = slice(0, 10)
+    categorical_feature_indexes = slice(10, 12)
```

Comment:
Especially when going through the lab with the students, it makes sense to explicitly have the columns listed out clearly.